### PR TITLE
Submission feature.

### PR DESCRIPTION
**Pull request by isht3**
- `evalai challenge CHALLENGE phase PHASE submit FILE`
- `evalai submission SUBMISSION`

Figuring out the proper way to build the submit command took some time.

### CODE REVIEW

File: evalai/challenges.py [hunk 1]
```diff
@@ -8,6 +8,14 @@
                                     get_future_challenge_list,
                                     get_challenge_count,)
 
+from evalai.utils.submissions import submit_file
+
+
+class Challenge(object):
+    def __init__(self, challenge=None, phase=None):
+        self.CHALLENGE = challenge
+        self.PHASE = phase
```
Comment (varunagrawal) [inline evalai/challenges.py]: Instance variable names should be lowercase.

File: evalai/utils/submissions.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    """
+    Pretty prints details of submission
+    """
+    team_title = "\n{}".format(style(submission['participant_team_name'], bold=True, fg="green"))
+    sid = "Submission ID: {}\n".format(style(str(submission['id']), bold=True, fg="blue"))
+
+    title = "{} {}".format(team_title, sid)
+
+    status = style("\nSubmission Status : {}\n".format(
+                                submission['status']), bold=True)
```
Comment (varunagrawal) [inline evalai/utils/submissions.py]: This should be of the form:
```python
format(submission['status']), 
       bold=True)
```

Similarly for the below statements.

**Comment by varunagrawal** [comment]
LGTM

**Comment by RishabhJain2018** [comment]
@isht3 Please resolve the merge conflicts.

**Comment by RishabhJain2018** [comment]
Please resolve the conflicts here.

**Comment by isht3** [comment]
@RishabhJain2018 Done!

**Comment by RishabhJain2018** [comment]
Please resolve conflicts here too!

**Pull request closed without merging**